src/components/index.ts

```diff
@@ -2,5 +2,5 @@ export * from "./Banner";
 export * from "./Button";
 export * from "./Spinner";
 export { default as Icon } from "./Icon";
-export * from "./Card";
+export { default as Card } from "./Card";
```

Comment:
Why not just the following?
```suggestion
export * from "./Card";
```

Reply:
Fixed with commit [43be1e8](https://github.com/Major-Eth/yearn-web-lib/pull/27/commits/43be1e8f6f63c8ddb4da56221609692ca8630770) !

Reply:
I'm not sure why we need to do it in such a complicated manner, could we just have

```typescript
// src/components/Card/index.ts
export { Card } from "./Card";
```

then

```typescript
// src/components/Card/index.ts

import { CardHead } from "./CardHead";
import { CardStats } from "./CardStats";

// ... export const Card = ({}) 

Card.Head = CardHead;
Card.Stats = CardStats;
```

Reply:
https://github.com/yearn/yearn-web-lib/pull/27/commits/5e31e5e27991eb2ff5556d0c94765633646d3e82